src/main/java/uk/gov/cabinetoffice/csl/domain/User.java

```diff
@@ -52,4 +52,8 @@ public String getOrganisationName() {
         return departmentHierarchy.stream().findFirst().map(BasicOrganisationalUnit::getName).orElse("");
     }
 
+    public boolean hasLineManager() {
+        return lineManagerEmail != null && lineManagerName != null;
```

Comment:
Can we use org.apache.commons.lang3.StringUtils.isBlank() for checking the null or empty string value?